readme.md

```diff
@@ -155,11 +156,15 @@ Marker `VALUE` specifies a property value e.g.: `red` => `[VALUE, 'red']`.
 
 #### Values list
 
+`[VALUE, 'red']+`
+
 A comma separated list of simple values e.g.: `red, green` => `[VALUE, 'red'], [VALUE, 'green']`.
 
 
 #### Compound value
 
+`[COMPOUND_VALUE_START], [VALUE, <value>]+, [COMPOUND_VALUE_END]`
```

Comment:
> I'm not sure why we'd need to handle space-separated compound variables any differently to anything else?

The idea was to have each value simple separately. Some of them may need vendor for e.g. prefixing or any other kind of post-processing. If we don't, we will need to split them up at parse time and loose perf.

> However, comma-separated compound variables definitely seems like a good point to break, but I don't see that catered for anywhere yet.

Comma separated values are described just the line above, called  "Values list". I was thinking that values list is basically when one property has multiple values, so I went for this syntax. Alternatively we can use COMPOUND_VALUE_START and a *COMBINATOR marker, similar to selectors.